File: spring-boot-properties/src/test/java/de/larmic/properties/service/HelloPropertiesTest.java
package de.larmic.properties.service;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

import static org.assertj.core.api.Assertions.assertThat;

@SpringBootTest
class HelloPropertiesTest {

    @Autowired
    private HelloProperties properties;

    @Test
    void assertProperties() {
        assertThat(properties.getValue()).isEqualTo("test properties");
    }
}

File: spring-boot-properties-kotlin/src/test/kotlin/de/larmic/properties/service/HelloPropertiesTest.kt
package de.larmic.properties.service

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
internal class HelloPropertiesTest {

    @Autowired
    private lateinit var properties: HelloProperties

    @Test
    fun `assert properties`() {
        assertThat(properties.value).isEqualTo("test properties")
    }
}